core/src/main/java/org/apache/brooklyn/core/entity/AbstractApplication.java

```diff
@@ -132,12 +133,12 @@ public AbstractApplication setParent(Entity parent) {
     @Override
     protected void initEnrichers() {
         super.initEnrichers();
-        
+
         // default app logic; easily overridable by adding a different enricher with the same tag
         ServiceStateLogic.newEnricherFromChildren().checkChildrenAndMembers().addTo(this);
-        ServiceStateLogic.ServiceNotUpLogic.updateNotUpIndicator(this, Attributes.SERVICE_STATE_ACTUAL, "Application created but not yet started, at "+Time.makeDateString());
+        ServiceStateLogic.ServiceNotUpLogic.updateNotUpIndicator(this, Attributes.SERVICE_STATE_ACTUAL, Lifecycle.CREATED);
```

Comment:
Why change these, out of interest? They are now inconsistent with the calls in `stop()` (which pass "Application stopping" etc).